src/hotspot/share/utilities/moveBits.hpp

```diff
@@ -26,74 +27,156 @@
 #define SHARE_UTILITIES_MOVEBITS_HPP
 
 #include "metaprogramming/enableIf.hpp"
+#include "utilities/byteswap.hpp"
+#include "utilities/debug.hpp"
```

Comment:
STATIC_ASSERT is static_assert, but sets the second argument to the stringification of the first. static_assert only accepts single arguments in, I think, C++17. But since its removed here anyway, its no longer needed.